p2p/src/authenticated/connection/handshake.rs

```diff
@@ -204,7 +204,7 @@ mod tests {
     use futures::SinkExt;
     use x25519_dalek::PublicKey;
 
-    const TEST_NAMESPACE: &[u8] = b"_TEST_NAMESPACE_";
+    const TEST_NAMESPACE: &[u8] = b"test_namespace";
```

Comment:
Switched to `"test_namespace"` which is already used in the file `cryptography/src/lib.rs`